Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:14.04
+
+# Install Oracle JDK7
+RUN \
+  apt-get update && \
+  apt-get install -y software-properties-common git curl && \
+  add-apt-repository -y ppa:webupd8team/java && \
+  echo oracle-java7-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections && \
+  apt-get update && \
+  apt-get install -y oracle-java7-installer && \
+  rm -rf /var/cache/oracle-jdk7-installer && \
+  rm -rf /var/lib/apt/lists/*
+
+# Install Mesos (need native library)
+RUN \
+  apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E56151BF && \
+  echo deb http://repos.mesosphere.io/ubuntu trusty main > /etc/apt/sources.list.d/mesosphere.list && \
+  apt-get update && \
+  apt-get install -y mesos && \
+  rm -rf /var/lib/apt/lists/*
+
+# Build the scheduler
+RUN \
+  git clone https://github.com/mesos/kafka /opt/mesos-kafka && \
```

Comment:
If they have this Dockerfile then they have the repo, shouldn't/wouldn't the location of the /opt/mesos-kafka be a volume mounted in or something of the sort?